README.md

```diff
@@ -2,7 +2,7 @@
 
 #### Table of Contents
 
-1. [Module Description - What the module does and why it is useful](#module-description)
+1. [Module description - What the module does and why it is useful](#module-description)
```

Comment:
One last thing! I think we need to at least update the id in this link to reflect the change to the description heading.